plugins/tmux/tmux.plugin.zsh

```diff
@@ -60,8 +60,12 @@ if which tmux &> /dev/null
 	# Wrapper function for tmux.
 	function _zsh_tmux_plugin_run()
 	{
+		# We attach to existing or start new session
+		if [[ -n "${@[(r)attach]}" || -n "${@[(r)new-session]}" ]]
```

Comment:
Might also want to check for the `attach-session` and `new` command aliases.

Should this be picking up the `"ZSH_TMUX_FIXTERM` stuff too, so it matches the attach/new-session behavior of the other branches?

Reply:
It's possible to make this into a one conditional with `|`:
```zsh
if [[ -n "${@[(r)attach|attach-session|new|new-session]}" ]]
```